lib/services/firewall.pm

```diff
@@ -30,13 +30,13 @@ sub install_service {
 }
 
 sub susefirewall2_to_firewalld {
-    my $timeout = 180;
+    my $timeout = 360;
     assert_script_run('susefirewall2-to-firewalld -c',                                     timeout => $timeout);
-    assert_script_run('firewall-cmd --permanent --zone=external --add-service=vnc-server', timeout => $timeout);
+    assert_script_run('firewall-cmd --permanent --zone=external --add-service=vnc-server', timeout => 60);
     # On some platforms such as Aarch64, the 'firewalld restart'
     # can't finish in the default timeout.
-    systemctl 'restart firewalld', timeout => $timeout;
-    script_run('iptables -S', timeout => $timeout);
+    systemctl 'restart firewalld', timeout => 60;
```

Comment:
How about systemctl('restart firewalld', timeout =>60) ?

Reply:
Is this coding rule we should using currently?

Reply:
I don't think so, just a suggestion :)